docs/source/raft_bft_migration.md

```diff
@@ -0,0 +1,154 @@
+
+3. Prepare the ConsenterMapping. Ensure all nodes are part of the consenters. 
+
+Note: all the channels should receive the same BFT `Metadata` and `ConsentersMapping` configuration.
+
+4. Ensure all ordering service nodes are running the same version of Fabric, and that this version is v3.0.0 or greater.
+5. Ensure all peers are running at least v3.0.0 of Fabric. Make sure all channels are configured with the channel capability that enables migration.
+
+
+## Phase 1 - entry to maintenance mode
+Prior to setting the ordering service into maintenance mode, it is recommended that the peers and clients of the network be stopped.
+Leaving peers or clients up and running is safe, however, because the ordering service will reject all of their requests, their logs will fill with benign but misleading failures.
+
+The only field that has to be changed during this step is in the channel configuration at `/Channel/Orderer/ConsensusType`.
+In this step of the channel configuration update, only change the `State` from `STATE_NORMAL` to `STATE_MAINTENANCE`. 
+Do not change the `Type` or the `Metadata` field yet. Note that the Type should currently be Raft.
+
+After updating the `State`, verify that each ordering service node has entered maintenance mode on each of the channels. 
+This can be done by fetching the last config block and making sure that on each channel the `Type` is Raft, the `Metadata` matched the Raft metadata and the `State` is `STATE_MAINTENANCE`.
+
+If the channels have been updated successfully, the ordering service is now ready for backup.
+
+Note:
+While in maintenance mode, normal transactions, config updates unrelated to migration, and `Deliver` requests from the peers used to retrieve new blocks are rejected. 
+This is done in order to prevent the need to both backup, and if necessary restore, peers during migration, as they only receive updates once migration has successfully completed. 
+In other words, we want to keep the ordering service backup point, which is the next step, ahead of the peer’s ledger, in order to be able to perform rollback if needed. 
+However, ordering node admins can issue `Deliver` requests (which they need to be able to do in order to continue the migration process).
+
+
+## Phase 2 - backup 
+Shut down all Raft ordering nodes, create a backup of the file system of these servers. Then restart the ordering service nodes.
+
+
+## Phase 3 - switch to BFT in maintenance mode
+This step in the migration process is another channel configuration update for each channel. 
+In this configuration update, switch the `Type` to `BFT` while keeping the State in `STATE_MAINTENANCE`, 
+and fill in the `Metadata` configuration and the `ConsentersMapping`. 
+Keep to set the `Metadata` configuration identically on all channels and keep to set the `ConsenterMapping` with all orderers.
```

Comment:
Wording is not clear.

Reply:
This is a description of the configuration update that needs to be done with the data that should be filled.